robottelo/ui/discoveredhosts.py

```diff
@@ -8,11 +8,13 @@
 class DiscoveredHosts(Base):
     """Manipulates Discovered Hosts from UI"""
 
-    def search(self, host_name):
-        """Searches existing discovered hosts from UI"""
+    def navigate_to_entity(self):
+        """Navigate to Discovered Hosts entity page"""
         Navigator(self.browser).go_to_discovered_hosts()
-        strategy, value = locators['discoveredhosts.hostname']
-        return self.wait_until_element((strategy, value % host_name))
```

Comment:
Hmm. How did that search work at all? There is no call to `search_entity()` method. We need one test run to make sure that our logic will work here